prestodb/client.py

```diff
@@ -240,6 +239,16 @@ def __init__(
                 raise ValueError('cannot use authentication with HTTP')
             self._auth.set_http_session(self._http_session)
 
+        default_exceptions = (
+            PrestoRequest.http.ConnectionError,  # type: ignore
+            PrestoRequest.http.Timeout,  # type: ignore
+        )
+        try:
+            from requests_kerberos.exceptions import KerberosExchangeError
```

Comment:
Can we delegate that part to the `auth` module to avoid leaking the implementation of the authentication method? For example with:

```python
self._exceptions = HTTP_EXCEPTIONS + auth.get_exceptions()
```